final-project/styles/main.scss

```diff
@@ -2,12 +2,1164 @@
+      width: 50%;
+    }
+  }
+
+  .about {
+    padding: Rem(68px) Rem(5px);
+    background-image: none;
+    margin-bottom: 0;
+
+    &__content {
+      &__item {
+        width: 24%;
+      }
+    }
+  }
+
+  .data {
+    height: 100%;
+    
+    &__half {
+      height: Rem(350px);
+    }
+
+    &__description {
+      width: 100%;
+    }
+  }
+
+
+  .blog {
+    display: none;
+  }
+
+  .plan-life{
+
+    &__desc {
+      margin-bottom: Rem(10px);
+    }
+
+    &__title {
+      margin-bottom: Rem(10px);
+    }
+
+    &__buttons {
+      &--active {
+        width: 40%;
+      }
+
+      &--light{
+        width: 40%;
+      }
+    }
+  }
+
+  .footer {
+    padding-right: 0;
+    padding-left: 0;
+  }
+}
+
+@media (max-width: 768px) {
+
+  .nav {
+    display: none;
+
+    &__hamburger {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      padding: Rem(3px) 10% 0;
+
+      &__ingridient {
+        height: Rem(4px);
+        width: Rem(35px);
+        margin: Rem(3px) 0;
+        background-color: #333;
+        border: none;
+        border-radius: 20%;
+      }
+    }
+  }
+
+  .hero {
+    overflow: hidden;
+    &__image {
+      overflow: hidden;
+      width: auto;
+    }
+
+    &__title {
+      width: 75%;
+      text-align: center;
+    }
+    &__text {
+      display: none;
+    }
+  }
+
+  .clients {
+    padding: Rem(40px) Rem(20px);
+    height: auto;
+  
+    &__list {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      &__item {
+        display: none;
+        &:first-child {
+          display: block;
+        }
+      }
+    }
+  }
+
+  .plan {
+    padding: Rem(15px) Rem(16px);
+    grid-template-rows: 1fr 1fr;
+    grid-template-columns: initial;
+
+    &-image {
+      width: 100%;
+    }
+
+    &__half {
+      width: 100%;
+    }
+
+    &__image {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin-bottom: Rem(50px);
+    }
+
+    &__btn-container {
+      display: none;
+      &--sm {
+        display: flex;
+      }
+    }
+
+    &__btn {
+      width: 50%;
+    }
+  }
+
+  .data {
+    grid-template-rows: 1fr 1fr;
+    grid-template-columns: initial;
+    height: 100%;
+    
+    &__half {
+      height: Rem(350px);
+    }
+
+    &__description {
+      width: 100%;
+    }
+  }
+
+  .about {
+    padding: Rem(68px) Rem(20px);
+    background-image: none;
+    margin-bottom: 0;
+
+    &__content {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      grid-template-rows: 1fr 1fr;
+      &__item {
+        width: auto;
+        padding: 0 4%;
+      }
+    }
+  }
+
+  .blog {
+    display: none;
+  }
+
+  .plan-life{
+    width: 100%;
+
+    &__desc {
+      margin-bottom: Rem(10px);
+    }
+
+    &__title {
+      margin-bottom: Rem(10px);
+    }
+
+    &__buttons {
+      &--active {
+        width: 40%;
+      }
+
+      &--light{
+        width: 40%;
+      }
+    }
+  }
+}
```

Comment:
Looks good overall, but it would be good if the scss was on different files for each section, would be easier to read and mantain (no need to change this, just a recommendation)